app/views/users/logout.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2><%= _('Are you sure you want to logout?') %></h2>
+<%= button_to _('yes'), logout_users_url, :class => 'btn btn-primary' %>
```

Comment:
'Yes' (capital Y) - I would also add a 'No' option, which goes back to the previous page?